modules/books/client/ListBooks/list-books.client.controller.js

```diff
@@ -0,0 +1,60 @@
+(function () {
```

Comment:
we need to make sure we add the IIFE as grunt task later, so you don't have to keep repeating it over and over again.